content/gophergala/data-mining.md

```diff
@@ -0,0 +1,154 @@
++++
+author = ["Marcel Hauf"]
+date = "2015-02-02T17:10:37+01:00"
+linktitle = "data-mining"
+title = "Data mining with Go"
++++
+
+# What is the idea behind Gophergala project heatingeffect
```

Comment:
Spell out "Gopher Gala" the way it is on their website. http://www.gophergala.com/

Add a question mark on the end?